<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>npcEditWidget</class>
 <widget class="QWidget" name="npcEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="saveChangesButton">
       <property name="text">
        <string>Save changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveNewNPCButton">
       <property name="text">
        <string>Save new NPC</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>238</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_1" columnstretch="1,1">
         <item row="0" column="1">
          <widget class="QLineEdit" name="identifierLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="currentXPLabel">
           <property name="text">
            <string>Current XP</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="identifierLabel">
           <property name="text">
            <string>Identifier</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="levelSpinBox">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="currentXPSpinBox">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="levelLabel">
           <property name="text">
            <string>Level</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="nextXPLabel">
           <property name="text">
            <string>Next XP</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Hitpoints</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="currentHPSpinBox">
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="maxHPSpinBox">
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="nextXPSpinBox">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="modelLabel">
           <property name="text">
            <string>Model</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
           <item>
            <widget class="QLineEdit" name="modelLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="modelPushButton">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveNewNPCButton</sender>
   <signal>clicked()</signal>
   <receiver>npcEditWidget</receiver>
   <slot>saveNewNPC()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveChangesButton</sender>
   <signal>clicked()</signal>
   <receiver>npcEditWidget</receiver>
   <slot>saveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>npcEditWidget</receiver>
   <slot>selectModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveNewNPC()</slot>
  <slot>saveChanges()</slot>
  <slot>selectModel()</slot>
 </slots>
</ui>
